### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by NIXKnight**

### CODE REVIEW

**Review by tasawernawaz — APPROVED** [review]
Looks good, only you can tell us if everything works as expected of not.

**Comment by NIXKnight** [comment]
The login page works and I can see 2 links after login. Not sure of the rest of the application.

File: README.md [hunk 1]
```diff
@@ -7,94 +7,201 @@ A third party tool which enables an author to record coding tasks by using an in
...
+```text
+prefix = ${HOME}/.npm
+```
+
+For the above environment changes to be effective:
+```console
+source ~/.bashrc
+```
+
+## **Setup CodeCast**
```
Comment (HamzaIbnFarooq) [inline README.md:68]: FYI: I think we have skipped Codecast's installation here e.g: `npm install, npm run build, etc`, will add that in a new commit

**Review by HamzaIbnFarooq — APPROVED** [review]
LGTM, @NIXKnight kindly merge this after having a look at my commit.

**Comment by HamzaIbnFarooq** [comment]
> The login page works and I can see 2 links after login. Not sure of the rest of the application.

If it's running, you can send me the link and credentials I will test that.

**Pull request merged**